### DISCUSSION TITLE
Clear the filter in stencil

### DISCUSSION THREAD

**Discussion by kvinita** [clientIO/joint#2583] [Q&A]
### Introduction

On a certain event, I want to clear the keyword in the filter of the stencil so that all items are loaded without being filtered.
How to programmatically clear the filter in stencil?
![image](https://github.com/clientIO/joint/assets/124018900/9d2c27ac-c7ac-4733-80fa-bc6dbb3dfcfc)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
This should work.
```js
stencil.filter(''); // reset filter
stencil.el.querySelector('input').value = ''; // clear the input
```
